docs/03-dev-howtos/01-ab-testing.md

```diff
@@ -6,26 +6,35 @@ Most tests can be written in JavaScript, although we can serve variants via Varn
 
 Read this if you want to [write a server-side AB Test](#write-a-server-side-test)
 
-# Guide
+## Guide
 
 There are six steps in the test lifecycle:
 
- - [Adding a switch to turn the test on & off](#adding-a-switch)
- - [Writing a test, which is typically a simple JS module](#writing-a-test)
- - [Running the test](#running-the-test)
+ - [Adding a switch to turn the test on & off)](#adding-a-switch). For server-side tests, this is included in the test definition.
+ - Writing a test e.g. [client-side](#writing-a-test) or [server-side](#write-a-server-side-test)
+ - [Running the test](#running-the-test) or [configuring your server-side test](#configure-the-test)
  - [Analysis of the test data](#analysis-of-the-test-data)
  - [Share your findings](#share-your-findings)
  - Delete the test
 
 ## Quick Tips
 
+- Decide if you need to create a [server-side AB test](#write-a-server-side-test) or a [client-side](#client-side-ab-tests) one. 
+  - You should consider testing server-side if you:
+    - Are making **rendering** changes (e.g. in DCR) and might be concerned about layout shift during rendering
```

Comment:
Minor but this is more of a pointer on why you would _implement_ server-side rendering rather than server-side testing. 

```suggestion
    - Are testing server-side **rendering** changes (e.g. in DCR)
```